backend.go

```diff
@@ -143,3 +182,15 @@ func formatJSONHeader(ctx context.Context, req *http.Request) {
 	bearerToken := "placeholder"
 	req.Header.Set("Authorization", bearerToken)
 }
+
+// FormatURLPath is a convenience method for ensuring that paths are properly
+// formatted
+func FormatURLPath(format string, params ...string) string {
+	// Convert parameters to interface{} and URL-escape them
+	untypedParams := make([]interface{}, len(params))
+	for i, param := range params {
+		untypedParams[i] = interface{}(url.QueryEscape(param))
+	}
+
+	return fmt.Sprintf(format, untypedParams...)
```

Comment:
Is the interface{} here because there maybe other types later on? 
Why cannot it be just []string instead of []interface{}?

Reply:
Good call. Will change to string

Reply:
Never mind - Sprintf takes an interface type and kicks a type error for an array of strings

Reply:
Oh right sorry, I am on 1.18beta and that is probs why it worked for me